src/website/views/components/breadcrumb.pug

```diff
@@ -1,4 +1,158 @@
 ---
 title: Breadcrumb
-disabled: true
 ---
+
+p.-text Breadcrumb indicate the current page’s location within a navigational hierarchy.
+
+h3 Examples
+p.-text
+  | To render a breadcrumb, use the class <code>a-breadcrumb</code>.
+  | A contextual class (e.g. <code>-separatorIcon</code>) is required to provide alternative separator styling.
+
+h4 Base
+.example.-mb3
+  .-p1
+    nav.a-breadcrumb.-text
+      ol
+        li.a-breadcrumb__item
+          a(href='#') Parent
+        li.a-breadcrumb__item
+          a(href='#') Child 1
+        li.a-breadcrumb__item.-active
+          | Child 2
+  :code(lang="html")
+    <nav class="a-breadcrumb -text">
```

Comment:
Let's render the base version at `font-size: .875rem;` so no `-text` class is needed on the base.

Reply:
If i put  font-size: .875rem; in the breadcrumb.scss base later the sizes modifers don't works

Reply:
@davidojedacl set -text in <nav>, as you had in the first version, so we can move on.